minetest.conf.example

```diff
@@ -30,24 +34,29 @@
 # Whether lavacooling should be enabled.
 #enable_lavacooling = true
 
-# Whether the stuff in initial_stuff should be given to new players
+# Whether the stuff in initial_stuff should be given to new players.
 #give_initial_stuff = false
-#initial_stuff = default:pick_steel,default:axe_steel,default:shovel_steel,default:torch 99,default:cobble 99
+#initial_stuff = default:pick_steel,default:axe_steel,default:shovel_steel,
+default:torch 99,default:cobble 99
 
-# Whether the TNT mod should be enabled
+# Whether the TNT mod should be enabled.
 #enable_tnt = <true in singleplayer, false in multiplayer>
 
-# The radius of a TNT explosion
+# The radius of a TNT explosion.
 #tnt_radius = 3
 
 # Enable the stairs mod ABM that replaces the old 'upside down'
 # stair and slab nodes in old maps with the new param2 versions.
 #enable_stairs_replace_abm = false
 
-# Whether you allow respawning in beds
-# Default value is true
+# Whether to allow respawning in beds.
+# Default value is true.
 #enable_bed_respawn = true
 
-# Whether players can skip night by sleeping
-# Default value is true
+# Whether players can skip night by sleeping.
+# Default value is true.
 #enable_bed_night_skip = true
+
+# If true, the engine's spawn search is used that does not check for a
+# suitable starting biome.
```

Comment:
For consistency, better as:
Whether the engine's spawn search, which does not check for a suitable starting biome, is used.
Default value is false.
(this applies for settingtypes.txt as well.)

Reply:
Ok, that reads better, i couldn't get the words right.